person.rb

```diff
@@ -0,0 +1,28 @@
+class Person
+  # Getter for id
+  attr_reader :id
+
+  # Getter and Setter for name and age
+  attr_accessor :name, :age
+
+  def initialize(age, name = 'Unknown', parent_permission: true)
+    @id = Random.rand(1..1000)
+    @name = name
+    @age = age
+    @parent_permission = parent_permission
+  end
+
+  # Return true if @age is greater or equal to 18 and false if not
+  def of_age?
+    return true if @age >= 18
+
+    false
+  end
+
+  # Make :of_age method private
+  private :of_age
+
+  def can_use_services
```

Comment:
![cus](https://github.com/jecode93/oop-school-library/assets/89055502/20913619-22e0-43d7-9c35-91a0cf6f37b5)
- [ ] As shown in the project requirement above, you are using the wrong name for your method, it should be `can_use_services?` not `can_use_services`, please kindly correct this and resubmit for approval :+1: